### PR TITLE
DHFPROD-8824: Document the Twizzlers API in Swagger

### PR DESCRIPTION
**Pull request by AkshayBajajML**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- N/A Added Tests
  

- ##### Reviewer:

- N/A Reviewed Tests
- N/A Added to Release Wiki

### CODE REVIEW

**Review by iveyMarklogic — APPROVED** [review]

File: marklogic-data-hub-central/ui-custom/openapi/openapi-spec.yaml [hunk 1]
```diff
@@ -0,0 +1,963 @@
...
+      summary: "fetches all the entities"
+      parameters:
+      - name: "userid"
+        in: "header"
+        required: true
+        schema:
+         type: string
+      responses:
+        "200":
+          description: "Response Ok"
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui-custom/openapi/openapi-spec.yaml:145]: The response for /api/explore/getEntityModels looks like this:
```
{
  "entityTypeNames": [
    "person",
    "organization"
  ]
}
```

File: marklogic-data-hub-central/ui-custom/openapi/openapi-spec.yaml [hunk 2]
```diff
@@ -0,0 +1,963 @@
...
+      tags:
+      - "Landing Page"
+      summary: "Returns requested metrics over particular period of time"
+      parameters:
+      - name: "userid"
+        in: "header"
+        required: true
+        schema:
+         type: "string"
+      - name: "mertics"
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui-custom/openapi/openapi-spec.yaml:161]: Fix typo: "metrics"

File: marklogic-data-hub-central/ui-custom/openapi/openapi-spec.yaml [hunk 3]
```diff
@@ -0,0 +1,963 @@
+openapi: 3.0.0
+info:
+  description: "This is a YAML file for API specifications."
+  version: "1.0.0"
+  title: "Twizzlers API Documentation"
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui-custom/openapi/openapi-spec.yaml:5]: Let's title it: "Entity Viewer API Documentation"

**Review by wooldridge — CHANGES_REQUESTED** [review]
@AkshayBajajML, this looks great. I added comments to generally address the following:

- For summaries, make wording consistent (e.g., "Returns...").
- Fix some of the params and payloads.
- Remove some config info that isn't used/needed.

**Comment by AkshayBajajML** [comment]
@wooldridge Thanks for reviewing it. I have made all of the mentioned changes and PR is ready to be reviewed. 
Let me know if there are any additional changes required.

**Comment by wooldridge** [comment]
The getEntityModels spec looks like this:
Screen Shot 2022-06-13 at 2 04 19 PM
The property name should be "entityTypeNames", here is the response:
```
{
  "entityTypeNames": [
    "person",
    "organization"
  ]
}
```

**Review by wooldridge — APPROVED** [review]
Looks good!!!

**Pull request merged**